src/txn/txn.c

```diff
@@ -495,6 +495,12 @@ __wt_txn_update_oldest(WT_SESSION_IMPL *session, uint32_t flags)
     uint64_t prev_last_running, prev_metadata_pinned, prev_oldest_id;
     bool strict, wait;
 
+    /*
+     * When not in strict mode we want to avoid scanning too frequently. Set a minimum transaction
+     * ID age threshold before we perform another scan.
+     */
+    const uint64_t non_strict_min_threshold = 100;
```

Comment:
This does not meet our style guidelines. Please move the declaration up (i.e. no blank line) and don't initialize in the declaration. You can leave the comment and initialization (but move it after the conn and txn_global settings below).

Reply:
done, thanks.